src/index.js

```diff
@@ -0,0 +1,176 @@
+require('dotenv').config();
+const inquirer = require('inquirer');
+
+const {
+  optionQuestions,
+  addDepartment,
+  addRole,
+  AddEmployee,
+  updateEmployee,
+} = require('./utils/questions');
+const initDatabase = require('./db');
+const {
+  allDepartmentsQuery,
+  allRolesQuery,
+  allEmployeesQuery,
+} = require('./utils/utils');
+
+const init = async () => {
+  // start asking the questions using inquirer
+  try {
+    //connect to the database
+    const { executeQuery, closeConnection } = await initDatabase({
+      host: process.env.DB_HOST,
+      user: process.env.DB_USER,
+      password: process.env.DB_PASSWORD,
+      database: process.env.DB_NAME,
+    });
+    let inProgress = true;
+
+    while (inProgress) {
+      const { journey } = await optionQuestions();
+      //check which option clicked and prompt the questions required
+      if (journey === 'view all departments') {
+        //get the data
+        const allDepartments = await executeQuery(allDepartmentsQuery);
+
+        // show the data
+        console.table(allDepartments);
+      }
+      if (journey === 'view all roles') {
+        //get the data
+        const allRoles = await executeQuery(allRolesQuery);
+        // show the data
+        console.table(allRoles);
+      }
+      if (journey === 'view all employees') {
+        //get the data
+        const allEmployees = await executeQuery(allEmployeesQuery);
+        // show the data
+        console.table(allEmployees);
+      }
+      if (journey === 'add a department') {
+        //   get the answer
+        let addDepartments = await addDepartment();
+        // get data
+
+        // get max id
+        const maxId = await executeQuery(
+          'SELECT max(id) as max FROM DEPARTMENT'
+        );
+        const newId = maxId[0].max + 1;
+
+        // store the data
+
+        const addAnotherDepartment = await executeQuery(
+          `INSERT INTO DEPARTMENT (id, department_name) VALUES ` +
+            `(${newId}, '${addDepartments.departmentName}')`
+        );
+        // show the data
+      }
+      if (journey === 'add a role') {
+        // get all departments
+        let allDepartmentNames = await executeQuery(
+          'SELECT department_name FROM DEPARTMENT'
+        );
+        //create the array of choices
+        let departmentArray = [];
+        const pushToArray = allDepartmentNames.map((item) =>
```

Comment:
If you are not going to use the variable further on in the code, you do not need to create one. Here, you could just do the map without storing it in the `pushToArray` variable.
```suggestion
       allDepartmentNames.map((item) =>
```